src/main/java/com/arpnetworking/metrics/mad/configuration/AggregatorConfiguration.java

```diff
@@ -337,6 +375,14 @@ public Builder setAkkaConfiguration(final Map<String, ?> value) {
         @NotEmpty
         private String _monitoringService = "mad";
         @NotNull
+        private ImmutableList<JsonNode> _monitoringSinks;
+        @NotEmpty
+        private String _metricsClientHost;
```

Comment:
Should this and `metricsClientPort` be marked as `@Nullable` now? The `ofNullable` above isn't correct upon inspection without this

Reply:
Done.